samples/ChatSample/ChatSample.csproj

```diff
@@ -1,11 +1,15 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
     <TargetFramework>netcoreapp1.1</TargetFramework>
     <PreserveCompilationContext>true</PreserveCompilationContext>
     <OutputType>Exe</OutputType>
     <UserSecretsId>aspnet-ChatSample-f11cf018-e0a8-49fa-b749-4c0eb5c9150b</UserSecretsId>
     <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8</PackageTargetFallback>
+    <RuntimeFrameworkVersion>1.2.0-*</RuntimeFrameworkVersion>
+    <IsPackable>false</IsPackable>
```

Comment:
@davidfowl IsPackable is true by default on all project types. Not sure why NuGet decided to make it that way. 

Setting this to false means calling "dotnet pack" on the .sln file won't try to pack this project.